### PR TITLE
feat(module:select): Select responsive max tag count

### PR DESCRIPTION
**Pull request by anddrzejb**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
Adds missing `MaxCountTag`, `MaxTagPlaceholder` and `MaxTagTextLenght`

### 💡 Background and solution
The responsive solution is based on js `window.resize` event. Originally I planned to use resize observer API, but because that change will have to span on more than a single functionality, I plan to make a separate PR that will apply a switch to that API (if successful). 

Current drawback is that `window.resize` is not really very responsive and takes some time to react.
Also, due to issues with `focus`, I changed current implementation to mirror what dotnet team had done with thier `ElementRefernece.FocusAsync()` built-in method. In fututre PR I will propose a global change on the focus method depending on the framework version used.

I am posting this to `master`, because this is bringing the `Select` component in line with antD. Please let me know if this should be in the `Feature` branch.

Also - please check API Chinese translation. I copied what I could from antD, but there are some comments from me in English.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |   Added missing `MaxCountTag`, `MaxTagPlaceholder` and `MaxTagTextLenght`        |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: site/AntDesign.Docs/Demos/Components/Select/demo/maxTagResponsive.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      }}
+    >
+      <Select {...selectProps} />
+      <Select {...selectProps} disabled />
+    </Space>
+  );
+};
+
+ReactDOM.render(<Demo />, mountNode);
+```
```
Comment (ElderJames) [inline site/AntDesign.Docs/Demos/Components/Select/demo/maxTagResponsive.md:57]: Please delete the jsx

**Comment by ElderJames** [comment]
This is very similar to the tabs navbar, but it hasn't been implemented correctly yet. 
Could you please help and see if you can repair it?

File: site/AntDesign.Docs/Demos/Components/Select/demo/maxTagResponsive.md (hunk 1 above)
Comment (anddrzejb) [reply site/AntDesign.Docs/Demos/Components/Select/demo/maxTagResponsive.md]: You were faster 😉

**Comment by anddrzejb** [comment]
I presume you are talking about the `Menu` component. I will have a look as soon as I finish with other tasks that I have planned.

**Comment by ElderJames** [comment]
No, I'm talking about the tabs component.

**Review by ElderJames — APPROVED** [review]

**Comment by anddrzejb** [comment]
Can you point me exactly to which part? The only similar functionality that I can see is in {slide section](https://antblazor.com/en-US/components/tabs#components-tabs-demo-slide) and it seems to me it behaves as it should (certainly behaves like the antD)...

**Comment by ElderJames** [comment]
Not be completed. Currently our implement can't scroll with the mouse wheel event, and can't handle the overflow items.

Please notice that the drop-down part is actually the part that is not shown on the tabs, not just the part that is overflowing from the tail.
ant-tabs-slide
.

**Comment by anddrzejb** [comment]
Ok, I think I get it now. Thanks for the explanation.

**Pull request merged**